doc/userguide/output/eve/eve-json-output.rst

```diff
@@ -69,6 +69,7 @@ Metadata::
             #payload-buffer-size: 4kb # max size of payload buffer to output in eve-log
             #payload-printable: yes   # enable dumping payload in printable (lossy) format
             #packet: yes              # enable dumping of packet (without stream segments)
+            #reference: no            # include reference information, if present, in the alert
```

Comment:
Should this be under the `rule` section just below? Thats where you enable adding the metadata from the rule.